Modules/_sqlite/connection.c

```diff
@@ -21,6 +21,12 @@
  * 3. This notice may not be removed or altered from any source distribution.
  */
 
+#ifdef HAVE_UNISTD_H
+#include <unistd.h>
+#else
+extern int unlink(const char *);
```

Comment:
This won't work under Windows. You have to use ``DeleteFileW``, see posixmodule.c